src/modules/git_commit.rs

```diff
@@ -437,22 +428,13 @@ mod tests {
             .output()?;
 
         create_command("git")?
-            .args(["tag", "v2"])
+            .args(["tag" , "v2"])
```

Comment:
This change is making cargo fmt fail is you change this to

```suggestion
            .args(["tag", "v2"])
```

It should pass